.github/workflows/push_pr.yaml

```diff
@@ -31,9 +32,43 @@ jobs:
           GOARCH: ${{ matrix.goarch }}
         run: |
           make compile
+
+  chart-lint:
+    name: Helm chart Lint
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: helm/chart-testing-action@v2.2.1
+
+      - name: Lint charts
+        run: ct --config .github/ct.yaml lint --debug --all
+
+      - name: Check for changed installable charts
+        id: list-changed
+        run: |
+          changed=$(ct --config .github/ct.yaml list-changed)
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+          fi
+      - name: Set up helm-unittest
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          helm plugin install https://github.com/quintush/helm-unittest
+      - name: Run unit tests
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
```

Comment:
Could we do it in the same step?

Like in the "original" one: https://github.com/newrelic/newrelic-k8s-metrics-adapter/pull/40/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR78-R87

Reply:
sure!